src/Easir/ruleset.xml

```diff
@@ -30,6 +30,7 @@
         <exclude name="SlevomatCodingStandard.Commenting.DocCommentSpacing.IncorrectAnnotationsGroup"/>
         <!-- Disallow unused use (ignored cause of issue when type from use is used only in static context -->
         <exclude name="SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse"/>
+        <exclude name="SlevomatCodingStandard.TypeHints.DeclareStrictTypes" />
```

Comment:
was this tested to?

Reply:
yes - it is not failing when declare is not present

Reply:
Is it passing when `declare(strict_types = 1);` is added?

Reply:
Yes - it is passing. The rule was just forcing it.